docs/source/graph_preprocessor.rst

```diff
@@ -0,0 +1,41 @@
+Graph-Based Feature Extraction
+==============================
+
+The GraphFeaturePreprocessor is a scikit-learn compatible preprocessing library that enables feature extraction from graph-structured data.  
+The library includes utilities for creating, updating, and transforming graph-structured data. 
```

Comment:
same comment about the library. I would say the GraphFeaturePreprocessor provides utilities for...